src/Physics/ini_model_DR.f90

```diff
@@ -297,6 +311,13 @@ SUBROUTINE DR_basic_ini(DISC,EQN,MESH,BND)                 ! global variables
       call rotateStressToFaultCS(EQN,MESH,DISC%Galerkin%nBndGP,nuc_xx,nuc_yy,nuc_zz,nuc_xy,nuc_yz,nuc_xz,EQN%NucleationStressInFaultCS,faultParameterizedByTraction)
       deallocate(nuc_xx,nuc_yy,nuc_zz,nuc_xy,nuc_yz,nuc_xz)
     end if
+
+    if (EQN%FL == 33) then !ImposedSlipRateOnDRBoundary
+        allocate(EQN%NucleationStressInFaultCS(DISC%Galerkin%nBndGP,6,MESH%Fault%nSide))
+        call rotateStressToFaultCS(EQN,MESH,DISC%Galerkin%nBndGP,nuc_xx,nuc_yy,nuc_zz,nuc_xy,nuc_yz,nuc_xz,EQN%NucleationStressInFaultCS, .TRUE.)
```

Comment:
Are you sure that this is correct? Vectors are not rotated in the same way as tensors!

Reply:
ha yes indeed, wrong.

Reply:
Ha wait, no, it is correct, because faultParameterizedbyTraction is set True, and therefore it uses create_fault_rotation_matrix function, which rotates vectors and not a tensor.

Reply:
No that is not true. create_fault_rotation_matrix also returns a transformation matrix for a tensor and not for a vector. Moreover, Stress is still multiplied with iT.

Reply:
What the function rotateStressToFaultCS does if faultParameterizedbyTraction=True is the following: The input stress tensor is given in a coordinate system with directions normal, strike, dip. The input stress tensor is then transformed to the physical space (x,y,z) and afterwards transformed to a fault-aligned space (with the same normal but different tangents!).

Reply:
ok, you are right. I will fix it.